bib/internal/container/container.go

```diff
@@ -83,9 +83,14 @@ func (c *Container) Stop() error {
 	if output, err := exec.Command("podman", "stop", c.id).CombinedOutput(); err != nil {
 		return fmt.Errorf("stopping %s container failed: %w\noutput:\n%s", c.id, err, output)
 	}
-	// when the container is stopped by podman it will not honor the "--rm"
-	// that was passed in `New()` so manually remove the container here.
+	// when the container is stopped by podman it may not honor the "--rm"
+	// that was passed in `New()` so manually remove the container here if it is still available
```

Comment:
Thanks for finding and fixing this! I wonder if it could be it's own PR though? Partly because it's orthogonal to the PR title, partly because I would love to poke at this a bit more and see if there is a way that avoids reading the error message (I don't think podman does i18n for errors but it might some day). But maybe I'm overthinking this. Alternatively we can of course keep as is and just do followups.

Reply:
I'd rather keep it as part of this since it lets the tests pass, at least locally for me on Fedora 39.